src/node_crypto.h

```diff
@@ -757,7 +757,7 @@ class PublicKeyCipher {
                      const void* oaep_label,
                      size_t oaep_label_size,
                      const unsigned char* data,
-                     int len,
+                     size_t len,
```

Comment:
Note: technically this change isn't strictly needed since we add the size check before this is called, but passing size_t len is more technically correct here and makes it easier for us later if we are able to lift the size restriction later for non-rsa ciphers.